crates/sui-cluster-test/src/cluster.rs

```diff
@@ -206,11 +206,10 @@ impl Cluster for LocalNewCluster {
                 rpc_server_url: indexer_address.as_ref().unwrap().ip().to_string(),
                 rpc_server_port: indexer_address.as_ref().unwrap().port(),
                 migrated_methods: IndexerConfig::all_migrated_methods(),
+                reset_db: true,
```

Comment:
Nice

Reply:
Suggestion: Maybe a todo here for when sui-test-validator is able to support data persistence?

Reply:
I think it should not, iiuc test cluster is an ephemeral service for testing etc. so it should always start fresh, but tbh the change here is just to make compiler happy, so I will leave the decision to @666lcz